src/Chapter1.hs

```diff
@@ -449,7 +449,7 @@ Implement the function that takes an integer value and returns the next 'Int'.
   function body with the proper implementation.
 -}
 next :: Int -> Int
-next x = error "next: not implemented!"
+next x = x+1
```

Comment:
Nice!

In Haskell all functions and operators are separated from its arguments by spaces 👾 
```suggestion
next x = x + 1
```